llvm/test/Transforms/LoopVectorize/AArch64/partial-reduce-dot-product-neon.ll

```diff
@@ -2108,6 +2108,97 @@ for.exit:                        ; preds = %for.body
   ret i32 %result
 }
 
+define i32 @zext_add_reduc_i8_i32(ptr %a) {
```

Comment:
should those tests go  in a new file? There's no dot product in the new tests?

Reply:
Done. I've also moved the non-dot-product tests from partial-reduce-dot-product.ll to a new partial-reduce.ll file.